parsing/parser.mly

```diff
@@ -2381,22 +2380,22 @@ optional_type_parameter:
   tys = separated_nonempty_llist(COMMA, optional_type_parameter)
     { tys }
 ;
-optional_type_variable:
+%inline optional_type_variable:
```

Comment:
This is unrelated to the inlining but the name `optional_type_variable` is really terrible. Would you mind breaking the separation of concerns for a minute and renaming it `ignorable_type_variable` or `type_variable_or_wildcard`?